zkevm-circuits/src/pi_circuit.rs

```diff
@@ -733,282 +1150,416 @@ impl<F: Field> PiCircuitConfig<F> {
         tx_id: usize,
         tx_id_next: usize,
         index: usize,
-        tx_value: F,
+        tx_value_bytes: &[u8],
```

Comment:
seems that we could just pass by a single `u8` if this function is only handling calldata.